ArrayOperations/Arrays.cs

```diff
@@ -0,0 +1,324 @@
+using System;
+
+namespace ArrayOperations
+{
+  /// <summary>
+  /// This class contain methods for array operations.
+  /// </summary>
+  public static class Arrays
+  {
+    /// <summary>
+    /// Sort an array using 'Bubble Sort' algorithm.
+    /// </summary>
+    /// <param name="input">Source array.</param>
```

Comment:
If you're going to do this commenting (you don't need to) then you should also include `<returns>`

Reply:
I just forgot to add it. I initially implemented bubble sort to do the sorting in place without returning another array.